doc/setup/email_gateway.rst

```diff
@@ -0,0 +1,40 @@
+:banner: banners/email_gateway.jpg
+
+==================
+Odoo email gateway
+==================
+
+The Odoo mail gateway is a better alternative than using an "incoming mail server"
+because the messages are received instantaneously.
+
+The principle is straightforward. Your incoming SMTP server will receive emails and directly forward
+them through a script (``odoo/addons/mail/static/scripts/odoo-mailgate.py``).
```

Comment:
Its principle is straightforward: your SMTP server executes the "mailgate" script for every new incoming email.